docs/src/content/docs/challenges/angular/4-context-outlet-typed.md

```diff
@@ -16,30 +16,34 @@ sidebar:
 
 ## Information
 
-Angular offer the static function [`ngTemplateContextGuard`](https://angular.io/guide/structural-directives#typing-the-directives-context) to strongly type structural directive.
+You can improve template type checking for custom directives by adding template guard properties to your directive definition. Angular offer the static function [`ngTemplateContextGuard`](https://angular.io/guide/structural-directives#typing-the-directives-context) to strongly type structural directives.
```

Comment:
offers

Reply:
i need to improve my english 😆

Reply:
There are plenty of grammar checkers and translators out there.  Check out [deepl](https://www.deepl.com/en/translator) and [quillbot](https://quillbot.com/).